tox.ini

```diff
@@ -138,6 +138,7 @@ commands =
 testpaths = tests
 filterwarnings =
     error
+    ignore::DeprecationWarning:docker.*:
```

Comment:
Nice. Could we maybe add a comment line with the link to the GitHub issue? So we can directly track if it's resolved.

Reply:
Done in 2510a38